### ISSUE TITLE
https://docs.prediction.io/support/ down

### ISSUE THREAD

**Issue by indrekpihor** [PredictionIO/PredictionIO#243]
Is this url coming back up anytime soon? Or will the SalesForce shut this site down for good?

**Comment by franklinkim** [comment]
This seems to affect the whole `https` site thus making a lot of internal links on the site useless.
You can still reach it through `http` (i.e. http://docs.prediction.io/support/)

**Comment by nqbao** [comment]
I hope this project won't become a deadend :(

**Comment by ravishchawla** [comment]
Is this issue also effecting engines? I get an SSL Handshake exception when trying to build my engine.

**Comment by dszeto** [comment]
Hi all, PredictionIO will continue as Apache PredictionIO (incubating). More details will follow this week. Sorry for the inconvenience meanwhile during the migration!

**Comment by dszeto** [comment]
@ravishchawla The build process should succeed. The SSL exception comes from a second thread that should not have interfered with the main build process.

**Comment by ravishchawla** [comment]
Yes, the build process did eventually complete, despite the error.

**Comment by CodyErekson** [comment]
@dszeto: Any more details yet? I have a project that is on hold until I can get a new build up and running.

**Comment by dszeto** [comment]
@CodyErekson Unfortunately I was not able to expedite the fix in infrastructure. I will instead drive towards a new patch and rely on Apache's infrastructure.

**Comment by CodyErekson** [comment]
@dszeto: Thanks for the update and new information.
